cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -729,7 +729,10 @@ subroutine evp (dt)
                            field_loc_NEcorner, field_type_vector)
       call grid_average_X2Y('U2TF',strocnxT)    ! shift
       call grid_average_X2Y('U2TF',strocnyT)
-
+      if (grid_system == 'CD') then
+          call grid_average_X2Y('E2UF',uvelE,uvel)
```

Comment:
Should these bee E2US and N2US?

Reply:
I concur.  I believe all averages in the CD-grid dynamics implementation should be "S" versions as the default until we decide otherwise.